ui/ducks/send/send.js

```diff
@@ -2731,9 +2732,7 @@ export function updateSendAsset(
 
           if (isCurrentOwner) {
             asset.error = null;
-            asset.balance = details.balance
-              ? addHexPrefix(details.balance)
```

Comment:
Using `addHexPrefix` is what triggered the bug in the NFT send flow.
When deploying a tokenId 1 and 2 with amount `10` and `15`
using `addHexPrefix` resulted in the balance being `0x10` and `0x15`.
In the `AssetBalanceText` component we will [calculate](https://github.com/MetaMask/metamask-extension/blob/56ca28472514fdb5141789d29744c728042b0e6c/ui/components/multichain/asset-picker-amount/asset-balance/asset-balance-text.tsx#L92) the number of tokens of nfts
`const numberOfTokens = hexToDecimal(asset.balance || '0x0');`
Which results in numberOfTokens being `16` and `21` instead of `10` and `15`